scholia/app/templates/chemical-class_publications-per-year.sparql

```diff
@@ -12,7 +12,7 @@ select ?year (count(?work) as ?number_of_publications) where {
       bind(year(?date) as ?year_)
       {
         select (min(?year_) as ?earliest_year) where {
-          { ?work wdt:P921/wdt:P31*/wdt:P279* target: . }
+          { ?work wdt:P921/wdt:P279* target: . }
           union { ?work wdt:P921/wdt:P361+ target: . }
           union { ?work wdt:P921/wdt:P1269+ target: . }
```

Comment:
Should we change the 3 lines to `wdt:P921 / (wdt:P31 | wdt:P279 | wdt:P361 | wdt:P1269)* target: ;

Reply:
I am not sure... how does ``wdt:P921 / (wdt:P31 | wdt:P279 | wdt:P361 | wdt:P1269)* target:` expand? Does that include mixing?

Reply:
It might be too expensive. In Synia I started to use ` ?work wdt:P921 / (wdt:P31 | wdt:P279 | wdt:P361 | wdt:P1269)? # Expensive: *
            target: ;`  here https://synia.toolforge.org/#topic/Q515701

Reply:
I know we have the UNION-like way all over, but to me "wdt:P921 / (wdt:P31 | wdt:P279 | wdt:P361 | wdt:P1269)*" looks prettier, e.g., why is there a P279 for P31 but not the others?